xml/art_sle_ha_pmremote.xml

```diff
@@ -774,31 +784,33 @@ Failed Actions:
    See the section <citetitle>Rolling Upgrade</citetitle> in the &haguide;
    <!--<xref linkend="sec.ha.migration.upgrade.rolling"/>-->
    for more information. It is available from <link
-    xlink:href="https://www.suse.coM/documentation/sle-ha-12"/>.
+    xlink:href="https://www.suse.com/documentation/sle-ha-12"/>.
   </para>
   <para>
    To upgrade your cluster and &pmrm; nodes, do the following:
   </para>
-  <para>
-   <remark>taroth 2017-08-09: is SLE really enough? I guess the cluster
-   nodes (and maybe also the remote nodes and guest nodes) also need the SLE HA
-   repos?</remark>
-  </para>
   <procedure>
    <step>
     <para>
-     For a rolling upgrade from &sle; 12 SP1 to 12 SP2:
+     For a rolling upgrade from &sle; and &productname; 12 SP1 to 12 SP2:
      Make sure you upgrade your remote nodes to the latest SP1 maintenance update.
     </para>
    </step>
    <step>
     <para>
-     Upgrade all cluster nodes to &sle; &productnumber; one by one.
+     Upgrade all cluster nodes to &sle; &productnumber; and &productname;
+     &productnumber; one by one.
     </para>
    </step>
    <step>
     <para>
-     Upgrade all &pmrm; nodes to &sle; &productnumber; one by one.
+     Upgrade all &pmrm; nodes to &sle; &productnumber; one by one and make sure
+     &productname; &productnumber; has been added as extension. Update the
+     packages <systemitem class="resource">pacemaker-remote</systemitem> and
+     <systemitem class="resource">crmsh</systemitem> as described in the
```

Comment:
If it's necessary to be more precise, probably something like:
"Update the packages pacemaker-remote, crmsh and there dependencies..."
?

(-- Installation of these two key packages actually introduced other dependencies, of which are from HA extension also, such as libqb, libpacemaker3, resource-agents and so on).